src/compass/utils/geo_metadata.py

```diff
@@ -265,3 +265,181 @@ def to_file(self, dst, fmt:str):
+                               'units': 'degrees'})
+
+        orbit_group = metadata_group.require_group('orbit')
+        self.orbit.save_to_h5(orbit_group)
+
+        add_dataset_and_attrs(metadata_group, 'orbit_direction',
+                              np.string_(self.orbit_direction),
+                              {'description':'direction of orbit (ascending, descending)'})
+
+        # VRT params
+        add_dataset_and_attrs(metadata_group, 'tiff_path',
+                              np.string_(self.tiff_path),
+                              {'description': 'path to TIFF containing the burst'})
+        add_dataset_and_attrs(metadata_group, 'i_burst', self.i_burst,
+                              {'description': 'Sentinel platform ID (S1, S2)'})
+        # window parameters
+        add_dataset_and_attrs(metadata_group, 'range_window_type',
+                              np.string_(self.range_window_type),
+                              {'description': 'name of the weighting window type used during processing'})
+        add_dataset_and_attrs(metadata_group, 'range_window_coefficient',
+                              self.range_window_coefficient,
+                              {'description': 'value of the weighting window coefficient used during processing'})
+
+        geogrid_group = metadata_group.require_group('geogrid')
+        add_dataset_and_attrs(geogrid_group, 'start_x', self.geogrid.start_x,
+                              {'description': 'starting y coordinate of geogrid output',
+                               'units': 'consistent with EPSG'})
+        add_dataset_and_attrs(geogrid_group, 'start_y', self.geogrid.start_y,
+                              {'description': 'starting y coordinate of geogrid output',
+                               'units': 'consistent with EPSG'})
+        add_dataset_and_attrs(geogrid_group, 'spacing_x',
+                              self.geogrid.spacing_x,
+                              {'description': 'x spacing coordinate of geogrid output',
+                               'units': 'consistent with EPSG'})
+        add_dataset_and_attrs(geogrid_group, 'spacing_y',
+                              self.geogrid.spacing_y,
+                              {'description': 'y spacing coordinate of geogrid output',
+                               'units': 'consistent with EPSG'})
+        add_dataset_and_attrs(geogrid_group, 'width', self.geogrid.width,
+                              {'description': 'width of geogrid output',
+                               'units': 'consistent with EPSG'})
+        add_dataset_and_attrs(geogrid_group, 'length', self.geogrid.length,
+                              {'description': 'length of geogrid output',
+                               'units': 'consistent with EPSG'})
+        add_dataset_and_attrs(geogrid_group, 'epsg',
+                              self.geogrid.epsg,
+                              {'description': 'EPSG of output geogrid'})
```

Comment:
```suggestion
                              {'description': 'EPSG code identifying the coordinate system used for processing,
```
Note that EPSG is unitless, so correct the units. Not sure if unitless is CF 1-8 compliant